### ISSUE TITLE
`side_panel` example fails to run (missing documentation); obvious workaround doesn't work.

### ISSUE THREAD

**Issue by stnbu** [mvlabat/bevy_egui#177]
With the `v0.20.2` tag checked out, and with (I think) reasonably recent Rust (both stable and nightly), I cannot run the `side_panel` example. I get the following error:

```
$ cargo run --example side_panel
   Compiling bevy_egui v0.20.2 (/Users/bob/git/bevy_egui)
error: missing documentation for a struct field
   --> src/lib.rs:611:10
    |
611 | #[derive(WorldQuery)]
    |          ^^^^^^^^^^
    |
note: the lint level is defined here
   --> src/lib.rs:1:9
    |
1   | #![deny(missing_docs)]
    |         ^^^^^^^^^^^^
    = note: this error originates in the derive macro `WorldQuery` (in Nightly builds, run with -Z macro-backtrace for more info)

error: could not compile `bevy_egui` due to previous error
```

I've tried deleting the top line of `src/lib.rs` and also replacing `deny` with `allow`. In both cases I do get the example to compile and run, but the side panel widgets are missing altogether. I just have tall empty rectangle on both sides of the scene. (otherwise looks normal).

This is on an x86 12.6.3 OSX with the following rust:

```
$ rustup show
Default host: x86_64-apple-darwin
rustup home:  /Users/bob/.rustup

installed toolchains
--------------------

stable-x86_64-apple-darwin (default)
nightly-x86_64-apple-darwin
1.52.1-x86_64-apple-darwin
1.60.0-x86_64-apple-darwin
1.61.0-x86_64-apple-darwin

installed targets for active toolchain
--------------------------------------

thumbv7m-none-eabi
wasm32-unknown-unknown
x86_64-apple-darwin

active toolchain
----------------

stable-x86_64-apple-darwin (default)
rustc 1.68.2 (9eb3afe9e 2023-03-27)
```

There's nothing "funny" in the repo:

```
$ git status
HEAD detached at v0.20.2
nothing to commit, working tree clean
```

**Comment by mvlabat** [comment]
Hey @stnbu! The issue with compilation is now fixed in the `main` branch.

Just to be clear, this is how the example is supposed to be looking:
![image](https://user-images.githubusercontent.com/2943388/233595910-2fc04dc3-eea8-44b8-a446-375d3dfb7e15.png)

I've added labels to make it clearer that it's indeed the intended behaviour (now in `main` too):
![image](https://user-images.githubusercontent.com/2943388/233596918-564ec02d-98de-4a4f-b132-e2a0ac50f33c.png)

Let me know if that answers your question.
